### ISSUE TITLE
Mandate PDF

### ISSUE THREAD

**Issue by craigjbass** [gocardless/gocardless-pro-java#3]
I noticed that the client does not allow the retrieval of Mandate PDF's

**Comment by matt-thomson** [comment]
We're working on adding this - I'll update here when this is available.

**Comment by craigjbass** [comment]
@matt-thomson cool!

**Comment by matt-thomson** [comment]
This is available in the newly released v0.2.3.  For example:

```java
try (InputStream pdf = gocardless.mandates().download("MD123").execute()) {
    // ...
}
```

**Comment by matt-thomson** [comment]
This now works differently in the new version of the API - see the [blog post](https://gocardless.com/blog/api-release-2015-07-06) for more background.

The API now returns the URL that can be used to access the mandate PDF:

```java
MandatePdf mandatePdf = goCardless.mandatePdfs().create().withLinksMandate(mandate).execute();
String mandateUrl = mandatePdf.getUrl();
```

You can then download the PDF from that URL, or issue a redirect to it.
